src/main/java/mate/academy/mapper/BookMapper.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.mapper;
+
+import mate.academy.dto.BookDto;
+import mate.academy.dto.CreateBookRequestDto;
+import mate.academy.model.Book;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.NullValueCheckStrategy;
+
+@Mapper(nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        implementationPackage = "<PACKAGE_NAME>.impl")
+public interface BookMapper {
+    @Mapping(target = "description", defaultValue = "undefined")
+    @Mapping(target = "coverImage", defaultValue = "undefined")
```

Comment:
what is the purpose of `defaultValue` here? migth these properties be missing in DB?

Reply:
Since these columns can be null, I just wanted to not display null in the response. But perhaps this is unnecessary). Removed these mapping annotations